action/recipes/config.go

```diff
@@ -20,7 +21,10 @@ import (
 )
 
 // ErrVerboseJSON is raised when JSONVerboseError can't find location of problem in JSON configuration file
```

Comment:
Can you move the comment for the error into the `var()` block over the error it is describing? Right now it looks like it is a description of both the `ErrVerboseJSON` and `ErrEnvVarUndefined` objects.